### PR TITLE
Signup form

### PR DESCRIPTION
**Pull request by Danneytrieu**

### CODE REVIEW

**Comment by jpatrici** [comment]
Can you add some description and the issue number to the PR? Also it'd be good to add a screenshot of what the component looks like. That way the reviewer knows what the PR is about

File: src/pages/SignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <RiLockPasswordLine />
+        <input
+          type="password"
+          placeholder="password"
+          {...register("password", { required: true })}
+        />
+        <button type="submit">SIGN UP</button>
+      </form>
+      <h3>Already have an account?</h3>
+      <a href="/sign-in">Login Here</a>
```
Comment (jpatrici) [inline src/pages/SignUp.tsx:43]: I think you can do `<Link>` from react router instead?
```
<Link to="/sign-in">Login Here</Link>
```
https://reactrouter.com/docs/en/v6/api#link

**Review by theodorusandi — APPROVED** [review]

**Comment by Danneytrieu** [comment]
Screen Shot 2022-04-21 at 11 13 32 AM

I play around with style a bit but will continue with styling after themeprovider is up running with figma 's color

File: README.md [hunk 2]
```diff
@@ -1,46 +1,15 @@
...
+Resouces:
 
-In the project directory, you can run:
+  -Design: https://www.figma.com/file/ojDUYgb1gmWJ9BTEVmdebp/Class-Management-%2F%2F-Chingu?node-id=0%3A1
+  
+  -Visual Sitemap: https://octopus.do/wpf8xbccze
+  
+Features — What are some key things your project can do?
 
-### `npm start`
```
Comment (jpatrici) [inline README.md:9]: I think `npm install` and `npm start` should still be in the README. Just some instruction on how to run the project locally

File: src/App.css [hunk 3]
```diff
@@ -1,2 +1,5 @@
 @import-normalize; /* bring in normalize.css styles */
```
Comment (theodorusandi) [inline src/App.css:2]: The GlobalStyle component just got merged in the last PR so we don't need app.css file. you can put font-family and other global settings inside global style file after normalize

**Review by theodorusandi — CHANGES_REQUESTED** [review]

**Comment by Danneytrieu** [comment]
Screen Shot 2022-04-21 at 5 07 36 PM
added a bit more style and separated the style components. I also resolved issues posted

File: src/styles/Section.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    font-size: 3rem;
+    cursor: pointer;
+  }
+  h2,
+  h3 {
+    font-size: 1rem;
+    font-weight: 300;
+  }
+  .link {
+    color: #20c997;
```
Comment (jpatrici) [inline src/styles/Section.tsx:24]: is this something that can be added to the theme file?

File: src/styles/Section.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  border-radius: 1.5rem;
+  width: fit-content;
+  height: auto;
+  padding: 2rem;
+  .tweeterIcon {
```
Comment (jpatrici) [inline src/styles/Section.tsx:14]:
```suggestion
  .twitterIcon {
```
nit 😄 idk if this is intentional? if it's intentional then dont mind me haha

File: src/styles/Container.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export const Container = styled.section`
```
Comment (jpatrici) [inline src/styles/Container.tsx:3]: i think this should just be a div?

**Review by jpatrici — COMMENTED** [review]
nice work on the styling 🎉 i left some comments as feedback around styled components. Remember that one thing that is useful about styled components, is that we can create small components that can be reused in other components/pages. It should be as independent as possible, so unless you think it's very specific to that component, avoid nesting the styles. 

I found this thread that I think gave a great example https://dev.to/titungdup/what-do-you-think-about-nested-styles-in-styled-components-3pdf

File: src/styles/Section.tsx (hunk 4 above)
Comment (Danneytrieu) [reply src/styles/Section.tsx:24]: yes! i hardcode it but once the theme provider is up, i will reimplement

File: src/styles/Section.tsx (hunk 5 above)
Comment (Danneytrieu) [reply src/styles/Section.tsx:14]: thanks for pointing that out =P

File: src/styles/Container.tsx (hunk 6 above)
Comment (Danneytrieu) [reply src/styles/Container.tsx:3]: i chose section over div for semantic purposes, and perhaps reuse it for dashboard's sections as well

**Comment by theodorusandi** [comment]
> Screen Shot 2022-04-21 at 5 07 36 PM
> added a bit more style and separated the style components. I also resolved issues posted

Nice! it looks neat, one thing tho, I think the icon is a little bit hard to see

File: tsconfig.json [hunk 7]
```diff
@@ -22,5 +22,5 @@
   },
   "include": [
     "src"
-  ]
+, "src/styles/AuthButton.tsx"  ]
```
Comment (jpatrici) [inline tsconfig.json:25]: @Danneytrieu why is this included in the tsconfig file?

File: src/pages/SignUp.tsx [hunk 8]
```diff
@@ -1,7 +1,66 @@
...
+              {...register("password", { required: true })}
+            />
+          </AuthField>
+
+          <AuthButton type="submit">SIGN UP</AuthButton>
+        </form>
+        <h3>Already have an account?</h3>
+        <Link className="link" to="/sign-in">
+          Login Here
+        </Link>
```
Comment (jpatrici) [inline src/pages/SignUp.tsx:60]: you can have this as a separate styled component
```
const StyledLink = styled(Link)`
   color: ....
   text-decoration: none
`;
```
and this will be
```
<StyledLink to="/sign-in"> Login Here </StyledLink>
```

File: tsconfig.json (hunk 7 above)
Comment (Danneytrieu) [reply tsconfig.json:25]: i have no idea how that ended up in there, i will remove it
